File: app/src/main/java/com/gmail/fattazzo/formula1world/news/reader/Formula1ComNewsReader.java
package com.gmail.fattazzo.formula1world.news.reader;

import org.jsoup.nodes.Element;

/**
 * @author fattazzo
 *         <p/>
 *         date: 20/07/17
 */
public class Formula1ComNewsReader extends RssNewsReader {

    @Override
    protected String getChannelImage(Element channelElement) {
        return "https://www.formula1.com/etc/designs/fom-website/icon128x128.png";
    }

    @Override
    protected String getNewsImage(Element newsElement) {
        return null;
    }
}

File: app/src/main/kotlin/com/gmail/fattazzo/formula1world/news/reader/Formula1ComNewsReader.kt
package com.gmail.fattazzo.formula1world.news.reader

import org.jsoup.nodes.Element

/**
 * @author fattazzo
 *
 *
 * date: 20/07/17
 */
class Formula1ComNewsReader : RssNewsReader() {

    override fun getChannelImage(channelElement: Element): String {
        return "https://www.formula1.com/etc/designs/fom-website/icon128x128.png"
    }

    override fun getNewsImage(newsElement: Element): String? {
        return null
    }
}
